src/main/java/com/puppycrawl/tools/checkstyle/api/AuditEvent.java

```diff
@@ -37,6 +37,7 @@
  *
  * @see AuditListener
  * @noinspection SerializableHasSerializationMethods
+ * @noinspectionreason we do not serialize class
  */
 public final class AuditEvent
     extends EventObject {
```

Comment:
I think we can update IDEA config of SerializableHasSerializationMethods
and add EventObject,Properties,Configuration to "Ignore subclasses of" to as we do not plan to do this completely.
but they are API classes, we should not change them withtout good reason.

Reply:
Done